app/renderer/components/tabs/tabPage.js

```diff
@@ -69,6 +70,15 @@ class TabPage extends React.Component {
     e.preventDefault()
   }
 
+  onContextMenu (e) {
+    e.stopPropagation()
+    windowActions.onTabPageMenu(this.props.index)
```

Comment:
just a nit, but I think `windowActions. onTabPageContextMenu` would be better here